src/components/Carousel.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+import './style/Carousel.scss';
+import f1 from '../img/frames/0.png';
+import f2 from '../img/frames/1.png';
+import f3 from '../img/frames/2.png';
+import f4 from '../img/frames/3.png';
+import f5 from '../img/frames/4.png';
+import f6 from '../img/frames/5.png';
+import f7 from '../img/frames/6.png';
+import f8 from '../img/frames/7.png';
+import f9 from '../img/frames/8.png';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlay } from '@fortawesome/free-solid-svg-icons';
+import { faPause } from '@fortawesome/free-solid-svg-icons';
+
+export default class AutoPlayMethods extends Component {
+  constructor(props) {
+    super(props);
+    this.play = this.play.bind(this);
+    this.pause = this.pause.bind(this);
+  }
+  play() {
+    this.slider.slickPlay();
+  }
+  pause() {
+    this.slider.slickPause();
+  }
+  render() {
+    const settings = {
+      dots: true,
+      fade: true,
+      infinite: true,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+      autoplay: true,
+      autoplaySpeed: 2000,
+      arrows: false
+    };
+    return (
+      <div className = "main">
+        <Slider ref={slider => (this.slider = slider)} {...settings}>
+          <div className = "frames">
+            <img src = {f1} className="frame" alt="first frame of filter calculation animation"></img>
```

Comment:
[Alttext](https://webaim.org/techniques/alttext/) here should be a little more descriptive and concise, and describe what each step is briefly.

Reply:
Since all the frames are showing basically the same thing, is it okay to give a complete description for the first image and then just say second example, third example, etc. for the following pictures?

Reply:
The idea is that you are trying to explain the process step by step through these images, right? The alttext should describe which step each image is conveying to the user. To better visualize this, here's a look at the carousel without images:

![carousel of alttext](https://user-images.githubusercontent.com/7704542/93183102-6b895380-f6ef-11ea-9835-107f24b03e3a.gif)

Your idea of describing the first image, then using something like "computing element 2,1" for the following alttext is a good idea. You don't need to be too verbose though, just a *brief* description of the operation is more than plenty.

Recall that a critical feature of alttext is that it is contextual. If you have already explained what "multiplying each element of a matrix" is at length, then you shouldn't need to do it again. Hope this answered your question!